### PR TITLE
[17.0][MIG] stock_picking_show_backorder: Migration to 17.0

### PR DESCRIPTION
**Pull request by RogerSans**
Standar migration
T-6444

@HaraldPanten 
@ValentinVinagre 
@luis-ron

### CODE REVIEW

File: stock_picking_show_backorder/views/stock_view.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
                 <page
                     name="backorders"
                     string="Backorders"
-                    attrs="{'invisible': [('backorder_ids','=',[])]}"
+                    invisible="backorder_ids == []"
```
Comment (Tisho99) [inline stock_picking_show_backorder/views/stock_view.xml]: i think: "not backorder_ids" is better

Comment (RogerSans) [reply stock_picking_show_backorder/views/stock_view.xml]: Changed. TNX

**Review by luis-ron — APPROVED** [review]
Functional Review: LGTM 👍🏻

File: stock_picking_show_backorder/README.rst [hunk 2]
```diff
@@ -42,8 +42,8 @@ Usage
 To use this module, you need to:
 
 1. Go to 'Inventory' and create a picking.
-2. Edit the 'Done' quantity of some operation line to be less than the
-   'To Do' quantity.
+2. Edit the 'Demand' quantity of some operation line to be less than the
+   'Quantity'.
```
Comment (Tisho99) [inline stock_picking_show_backorder/README.rst]: Its the opposite, the quantity must me less than the demand to create a backorder. You cant edit the demand

**Review by Tisho99 — CHANGES_REQUESTED** [review]

File: stock_picking_show_backorder/README.rst (hunk 2 above)
Comment (RogerSans) [reply stock_picking_show_backorder/README.rst]: Done. TNX

**Review by Tisho99 — APPROVED** [review]
LGTM. Functional and code review

**Review by HaraldPanten — APPROVED** [review]
@JordiBForgeFlow @LoisRForgeFlow 

Could we merge this one?

THX!

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

/ocabot merge nobump

**Pull request merged**